cwl/src/test/resources/three_step.cwl

```diff
@@ -1,48 +1,47 @@
 cwlVersion: v1.0
 class: Workflow
-id: file:///Users/danb/wdl4s/three_step.cwl
+id: three_step
```

Comment:
weren't the old id's more like what we'd expect with Salading?

Reply:
this was actually one of my comments in the original PR. It's ok because we're salading this file before using it anyway